sagemaker-remote-function/quick_start/requirements.txt

```diff
@@ -0,0 +1,2 @@
+pandas
+sagemaker>=2.148.0,<3
```

Comment:
I am not sure if <3 is really needed. I don't think we will knowingly push a backwards incompatible release, In future, when sagemaker version greater than three is released, we will have to come back and update these notebooks to make it compatible with latest versions.

Not blocking, your call if you still think this is better

Reply:
In the versioning systems, it is accepted that a major version may introduce backward incompatibility. We don't have any other functionality of the PySDK showcased in these examples. IMO, the <3 version restriction will make sure that our feature showcasing is not broken even with the major version bump.